### ISSUE TITLE
2nd part of part 3

### ISSUE THREAD

**Issue by lrbridge** [lrbridge/mp1#6]
We need to pick one of the optional pieces.  Thoughts on which?

**Comment by GoingtoRock** [comment]
I was thinking either to go with second one or third one, since I'm not good at graphics. From my thought, second one seems the easiest.

**Comment by GoingtoRock** [comment]
On second thought, creating ghosts would be easier. But let me think on the second one first.

**Comment by lrbridge** [comment]
Wait I'm confused.  I already did part 3 with the ghost?  Just want to make sure you knew that.  I saw one of your commits was about implementing the ghost, so I wanted to check to make sure we weren't duplicating work.

It seems like the first part of part 3 with ghosts is working pretty well @GoingtoRock

**Comment by lrbridge** [comment]
I haven't done any of the 3 choice options, so that's what I'm hoping you'll start on?

**Comment by GoingtoRock** [comment]
Yepp. Decided to do the multiple/advanced ghost thing, and I added the new ghost called FastGhost which basically moves twice as fast as the normal ghost. And for the last two test cases I added one more of the normal ghost - seems like it is working fine.

**Comment by lrbridge** [comment]
Is the AGhost just the same as the normal ghost, but now we just have 2 of them in the maze?

**Comment by GoingtoRock** [comment]
Yes. Turns out that I don't have to implement the unique ghost. So Aghost is acting same as the normal ghost. I just gave it a different letter, for easier tracking of individual ghost.
